tests/test_forgot_password.py

```diff
@@ -0,0 +1,43 @@
+from init import BasePage
+import unittest
+from pages.auth_page import AuthPage
+from pages.vacancies_page import VacanciesPage
+from config import EMAIL_SUBJECT_PASSW_RECOVERY, USERNAME_PASSW_RECOVERY, \
+    EMAIL_FORGOT_PASSWORD, FROM_FORGOT_PASSWORD, OLD_PASSWORD, NEW_PASSWORD
+from utilities.get_email import get_link
+from pages.forgot_password_page import ForgotPasswordPage
+from pages.confirm_password_page import ConfirmPassword
+
+
+class TestForgotPassword(BasePage):
+    @unittest.skip('skip, as password doesnt change')
+    def test_forgot_password(self):
+        login = AuthPage(self)
```

Comment:
please, use `__init__` in this class for such things, make something like this:
`def __init__(self):
    self.login = AuthPage(self)
    self.vacancies = VacanciesPage(self)
    self.forgot_password = ForgotPasswordPage(self)
    self.confirmation_password = ConfirmPassword(self)`
but don't do it in tests